sled-agent/src/bootstrap/agent.rs

```diff
@@ -425,7 +425,7 @@ impl Agent {
         let paths = sled_config_paths(&storage_resources).await;
         let maybe_ledger =
             Ledger::<PersistentSledAgentRequest>::new(&ba_log, paths).await;
-        let make_agent = move |initialized| Agent {
+        let make_bootstrap_agent = move |initialized| Agent {
```

Comment:
This was just a rename for clarity - I had to scan this code when debugging, and wanted to make it more clear.